src/service/bill_service.rs

```diff
@@ -256,19 +251,19 @@ impl BillService {
         postfix: &str,
     ) -> Result<String> {
         let identity_public =
-            IdentityPublicData::new(identity.identity.clone(), identity.peer_id.to_string());
+            IdentityPublicData::new(identity.identity.clone(), identity.node_id.to_string());
         let caller_identity_bytes = serde_json::to_vec(&identity_public)?;
         match other_party {
             None => Ok(format!(
                 "{prefix}{}{postfix}",
-                &util::base58_encode(&caller_identity_bytes)
+                &hex::encode(caller_identity_bytes)
```

Comment:
I think there was an issue when merging the latest master - we replaced hex encoding with base58